src/gui/runtime/plugin/nvim_gui_shim.vim

```diff
@@ -149,3 +149,32 @@ function GuiClipboard()
           \ }
 	call provider#clipboard#Executable()
 endfunction
+
```

Comment:
Usually in the shim user facing commands/functions are named as GuiSomething.

Reply:
I'll rename the commands then, I thought that it would be fine to namespace them elsewhere since it's different widgets.

Do you also want the <Plug> mappings to be `<Plug>GuiTreeviewName` ?